tests/http_layer/test_request_construction.py

```
@@ -0,0 +1,1286 @@
+        body = _extract_request_body(route.calls.last.request)
+        assert "StartVM" in body["query"]
+        assert body["variables"] == {"id": "vm-123"}
+        assert result["success"] is True
+
+    @respx.mock
+    async def test_stop_sends_mutation_with_id(self) -> None:
+        route = respx.post(API_URL).mock(
+            return_value=_graphql_response({"vm": {"stop": True}})
+        )
+        tool = self._get_tool()
+        result = await tool(action="stop", vm_id="vm-456")
```

Ruff (0.15.0)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)